test/check-ostree

```diff
@@ -659,7 +668,8 @@ class OstreeCase(MachineCase):
         rhsmcertd_hack(m)
 
         # preloading works, no updates available
-        self.login_and_go("/system")
```

Comment:
This seems like copy-pasta problem. Why it now goes to `/updates` when it used to go to `/system`? And it checks `Health` card so it seems incorrect.